src/platform/testing/e2e/cypress/support/commands/webComponents.js

```diff
@@ -0,0 +1,194 @@
+const FORCE_OPTION = { force: true };
+
+Cypress.Commands.add('fillTextWebComponent', (field, value) => {
+  if (value) {
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-text-input[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('input')
+      .as('currentElement')
+      .type(value);
+
+    cy.get('@currentElement').then(el => {
```

Comment:
this pattern of `.as('something')` then `cy.get('@something')` was recommended by eslint to make sure you're always getting the latest data.